src/main/webapp/pages/analysis/visualizations/phylocanvas-metadata.html

```diff
@@ -73,10 +73,10 @@
         <div class="spaced-bottom" id="phylocanvas"></div>
       </script>
       <script type="text/ng-template" id="metadata-component.tmpl.html">
-        <div class="spaced-bottom" ng-repeat="term in $ctrl.terms"> <input
```

Comment:
Docs pages need updated https://irida.corefacility.ca/documentation/user/user/analysis-visualizations/

Reply:
Updated in 4277646